src/components/TabsPage.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const TabsPage:React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+
+  const selectedTabText = () => {
```

Comment:
Let's rename the function name a little bit, to show that it's an action
```suggestion
  const getSelectedTabText = () => {
```